services/static-webserver/client/source/class/osparc/store/Functions.js

```diff
@@ -101,7 +98,10 @@ qx.Class.define("osparc.store.Functions", {
           curatedOrderBy.field = "created_at";
           break;
         case "name":
-          // stays the same
+          // Backend does not currently support sorting by 'name'.
+          // Fallback: sort by 'modified_at' instead.
+          // TODO: Remove this workaround once backend supports sorting by 'name'.
```

Comment:
Create an issue for the backend, if there isn't one and add it to the comments above.
Also mention for the backend people on what to change in order to remove this workaround.